src/erpe/pom.xml

```diff
@@ -91,10 +91,25 @@
       <artifactId>slf4j-log4j12</artifactId>
     </dependency>
 
+
+   <dependency>
+       <groupid>org.junit.jupiter</groupid>
+        <artifactid>junit-jupiter-api</artifactid>
+        <version>5.0.0-M3</version>
+        <scope>test</scope>
+    </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
+       <groupid>junit</groupid>
+       <artifactid>junit</artifactid>
+       <version>4.12</version>
+       <scope>test</scope>
+   </dependency>
```

Comment:
so this is required by juint 5? I've saw junit 5 example without junit 4 dependencies.